core/qthelper.cpp

```diff
@@ -1143,6 +1143,11 @@ QString localFilePath(const QString &originalFilename)
 	return localFilenameOf.value(originalFilename, originalFilename);
 }
 
+// the water types need to match the watertypes enum above
```

Comment:
omit `above` as `watertypes` is defined in qthelper.h?

Reply:
👍